lib/app.dart

```diff
@@ -13,17 +14,25 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return KeyboardDismisser(
-      child: ObserverBuilder<ConfigStore>(
-        builder: (context, store) => MaterialApp(
-          title: 'Liftoff',
-          supportedLocales: L10n.supportedLocales,
-          localizationsDelegates: L10n.localizationsDelegates,
-          themeMode: store.theme,
-          darkTheme: store.amoledDarkMode ? amoledTheme : darkTheme,
-          locale: store.locale,
-          theme: lightTheme,
-          home: const HomePage(),
-        ),
+      child:
+          // observer builder
```

Comment:
nit: if you move this comment on the same line as `ChangeNotifierProvider(` it'll fix this awkward line break (probably).